### PR TITLE
File generator

### PR DESCRIPTION
**Pull request by slavapa**

### CODE REVIEW

File: lib/file_utils/file_generator/README [hunk 1]
```diff
@@ -0,0 +1,43 @@
+File Generator utility
+------------------------------------------------------------------
+Generates files with random content with random file size according to the given FileGenerator Parameters.
+The FileGeneratorParameters Represents parameters for file generation and can be defined in the YAML as follow:
+defaults:
+  target_path:  C:/Users/kolman/test_files                  #Represents the target path for files generation
+  file_name_template: auto_generated_file_4_backup_server   #Represents the file name template for generated file
+  dir_name_template:  test_dir_4_backup_server              #Represents the directory name template for generated file
+  file_size_in_mb: 500                                      #Represents the required file size in MB. Not relevant if random calculation is triggered
```
Comment (kolmanv) [inline lib/file_utils/file_generator/README]: Break the comment line in the following way:
flag: value         # comment very logg...
                           # continue comment here.
flag2: value 2   # another comment.
Number of columns should be no more then 100. See style guide.
https://sites.google.com/site/rndbbnorth/cheat-sheet/code-styling

Comment (kolmanv) [inline lib/file_utils/file_generator/README]: Fix this and merge by yourself.

**Comment by kolmanv** [comment]
Please fix my comments and merge by yourself.

**Comment by slavapa** [comment]
I'll today.

-----Original Message-----

**Pull request merged**